src-ui/src/app/components/document-list/bulk-editor/bulk-editor.component.ts

```diff
@@ -340,6 +340,10 @@ export class BulkEditorComponent {
       } else {
         modal.componentInstance.message = $localize`This operation will remove the storage path from ${this.list.selected.size} selected document(s).`
       }
+      modal.componentInstance.message +=
+        '<br/><br/><em>' +
+        $localize`Note: changing the storage path does not apply changes to files until you have run the 'document_renamer' utility.` +
```

Comment:
That’s not correct. Bulk edit touches every file like you would edit it one by one. The renaming is applied directly.

Reply:
Oh I didnt realize that. Then yes I will remove this note. 

But for me this isnt working at the moment. When I bulk edit the files dont move until I run it.

Reply:
Than I have to check this. But I cannot do it today.

Reply:
Oh cool, never mind it was because my Q cluster wasn't running and this uses an async task. Ok great I'll remove that note.